spec/bulk_update_file_spec.rb

```diff
@@ -86,8 +85,6 @@ def call(**options)
     stub_govuk_repos(%w[foo])
     stub_github_repo("foo", feature_branches: [branch])
     stub_create_branch_request("foo", branch)
-    stub_create_contents_request("foo", path: file_path, content: file_content, commit_title: pr_title, branch:)
-    stub_create_pull_request("foo", branch:, title: pr_title, description: pr_description)
```

Comment:
I don't understand why these stubs are being removed - could you add some more detail to the `Returns PRs from stub` commit if you end up keeping it?